README.md

```diff
@@ -23,47 +23,56 @@ The router map should seem familiar from ember's router, with some differences.
 map might looks like for an app similar to Twitter:
 
 ```js
-tabRouter('tabs', [
-  stackRouter('timelineTab', [
-    route('timeline'),
-    route('tweet'),
-    route('profile'),
-    route('photos'),
-    route('video'),
-    // etc...
-  ]),
-  stackRouter('searchTab', [
-    route('trends'),
-    tabRouter('searchResultsTabs', [
-      stackRouter('topTab', [
-        route('tweet'),
-        route('profile'),
-        // etc...
-      ]),
-      stackRouter('latestTab', [
-        route('timeline'),
-        route('tweet'),
-        // etc...
+import {
+  mount,
+  route,
+  stackRouter,
+  tabRouter
+} from 'ember-navigator';
+
+this.mountedRouter = mount(
```

Comment:
This is in router.js? No export required?

Reply:
Good question. There is not currently a prescriptive or opinionated answer to this question in ember-navigator. I do it in a service in our app. @machty's demo has it in application controller. I'll give some thought to how to document this.